crates/floresta-cli/src/rpc.rs

```diff
@@ -64,7 +64,12 @@ pub trait FlorestaRPC {
     /// filters, we'll need to download the entire blockchain again, which will take a while.
     /// The rescan parameter is the height at which to start the rescan, and should be at least
     /// as old as the oldest transaction this descriptor could have been used in.
-    fn rescanblockchain(&self, start_height: u32) -> Result<bool>;
+    fn rescanblockchain(
+        &self,
+        start_height: u32,
+        stop_heigth: u32,
+        use_timestamp: bool,
```

Comment:
This makes them required for people using the rust code. If something is optional, we use an `Option` here